wp-content/themes/core/content/single/post.twig

```diff
@@ -1,32 +1,50 @@
 <article class="item-single">
 
-	{{ post.featured_image }}
-
-	<div class="item-single__content t-content">
-		{{ post.content }}
-	</div>
-
-	<footer class="item-single__footer">
-
-		<ul class="item-single__meta">
-
-			<li class="item-single__meta-date">
+	<div class="item-single__featured-image">{{ post.featured_image }}</div>
+
+	<div class="l-container__stagger item-single__content">
+		<h1 class="item-single__headline">{{ post.title }}</h1>
+		<div class="item-single__meta">
+
+			{% if post.categories %}
+				<div class="item-single__meta__categories">
+					<ul class="item-single__meta__categories__list">
+						{% for category in post.categories %}
+							<li class="item-single__meta__categories__list-item">
+								<a class="item-single__meta__categories__category-link" href="{{ category.permalink }}">
+									{{ category.name|esc_html }}
+								</a>
+							</li>
+						{% endfor %}
+					</ul>
+				</div>
+			{% endif %}
+
+			<div class="item-single__meta__date u-fc-nevada">
 				<time datetime="{{ post.time['c']|esc_attr }}">
 					{{ post.date|esc_html }}
 				</time>
-			</li>
+			</div>
 
-			<li class="item-single__meta-author">
-				{{ __('by')|esc_html }}
-				<a href="{{ post.author.url|esc_url }}" rel="author">
-					{{ post.author.name|esc_html }}
-				</a>
-			</li>
+		</div>
+		<div class="t-content">
+			{{ post.content }}
+		</div>
+		<footer class="item-single__footer">
 
-		</ul>
+			{{ include( 'components/share.twig' ) }}
 
-		{{ include( 'components/share.twig' ) }}
+		</footer>
+	</div>
 
-	</footer>
+	<div class="item-single__related-posts">
+		<hr class="item-single__related-posts__divider">
+		<h2 class="item-single__related-posts__section-title">More Posts</h2>
```

Comment:
We always make sure to namespace/localize any strings in our code so they are ready for translation out of the box: `{{ __( 'More Posts' )|esc_html }}`